pkg/collector/corechecks/servicediscovery/module/config.go

```diff
@@ -13,21 +13,45 @@ import (
 
 	sysconfig "github.com/DataDog/datadog-agent/cmd/system-probe/config"
 	ddconfig "github.com/DataDog/datadog-agent/pkg/config/setup"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
 )
 
 const discoveryNS = "discovery"
 
 type discoveryConfig struct {
 	cpuUsageUpdateDelay time.Duration
+	ignoreComms         map[string]struct{}
 }
 
 func newConfig() *discoveryConfig {
 	cfg := ddconfig.SystemProbe()
 	sysconfig.Adjust(cfg)
 
-	return &discoveryConfig{
+	conf := &discoveryConfig{
 		cpuUsageUpdateDelay: cfg.GetDuration(join(discoveryNS, "cpu_usage_update_delay")),
 	}
+	conf.loadIgnoredComms(cfg.GetString(join(discoveryNS, "ignored_command_names")))
```

Comment:
You can use something with a stronger type: `cfg.GetStringSlice`

Reply:
The `BindEnvAndSetDefault` becomes `[]string{...}` then.

Reply:
I've changed code to use slice of string to setup default values.